pyproject.toml

```diff
@@ -88,6 +88,13 @@ classifiers=[
     "torch",
     # fairseq is also a dependency, but it must be installed separately
   ]
+  asr_bleu = [
+    "pandas==1.4.3",
```

Comment:
Do we really need such precise versions?

Reply:
You're right, big libraries like pandas will be backwards compatible. I'll just keep the precise version for sacrebleu, as we've had issues before with libraries within the FAIR ecosystem needing to be specific versions. Resolved with https://github.com/facebookresearch/stopes/pull/52/commits/c422fc7985fe8c84f42b7829d3c2efaadf3eef9a